src/app/series/directives/series-podcast.component.html

```diff
@@ -21,9 +21,11 @@
           <button [publishCopyInput]="pubFeed">Copy</button>
           <a class="button" target="_blank" rel="noopener" [href]="podcast?.publishedUrl">Open Link</a>
         </publish-fancy-field>
-        <publish-fancy-field label="Preview Feed" small=1>
```

Comment:
I think it's being used for testing right now, so I was worried about removing support entirely. I was considering checking if we'd done anything with the `process.env.NODE_ENV` yet and log it out when ` !== 'production'`, but I didn't check.